sycl/test/check_device_code/matrix/matrix-nvptx-bfloat16-test.cpp

```diff
@@ -0,0 +1,200 @@
+// REQUIRES: cuda
+
+// RUN: %clangxx -fsycl-device-only -fsycl-targets=nvptx64-nvidia-cuda -Xsycl-target-backend --cuda-gpu-arch=sm_80 -DSYCL_EXT_ONEAPI_MATRIX=3 -S -Xclang -emit-llvm %s -o -| FileCheck %s
+
+#include <CL/sycl.hpp>
+
+using namespace sycl;
+using namespace sycl::ext::oneapi::experimental::matrix;
+using sycl::ext::oneapi::experimental::bfloat16;
+
+constexpr int stride = 16;
+
+int main() {
+
+  buffer<bfloat16, 1> bufA(nullptr, range<1>(1));
+  buffer<bfloat16, 1> bufB(nullptr, range<1>(1));
+  buffer<float, 1> bufC(nullptr, range<1>(1));
+  buffer<float, 1> bufD(nullptr, range<1>(1));
+
+  queue q;
+
+  q.submit([&](handler &cgh) {
+    auto accC = bufC.get_access<access::mode::read_write>(cgh);
```

Comment:
Write new tests in SYCL 2020 with simple accessor constructor

Reply:
Thanks, done.